src/web/modules/gateway_accounts/createSuccess.njk

```diff
@@ -19,24 +19,27 @@
 A {{ account.type | upper }} Gateway account has been created through Admin Users.
 </p>
 
-{% if isStripe %}
-	<h2 class="govuk-heading-m">What to do next</h2>
-	<p class="govuk-body">In Zendesk, setup the Stripe go live email using the Zendesk macro.  
-	<p class="govuk-body">Replace the placeholders in the macro with the following:</p>
+<h2 class="govuk-heading-m">What to do next</h2>
+<p class="govuk-body">In Zendesk, setup the Stripe go live email using the Zendesk macro.  
```

Comment:
Can we display it only for Stripe ?

Reply:
ah didn't realise it said Stripe here, I could just remove the word Stripe?

Reply:
yeah lets remove Stripe in the first instance. Could add a bit more to say which macro to use for Stripe and other payment providers. This could be added later. if needed.

Reply:
👍  I've amended